base/untar.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Download and extract tarball archive from the network.
+
+set -ex
```

Comment:
`-x` left over. If you'd like to track progress, we should add progress messages, IMO.
Would it make sense to add `-u` here as well?

Reply:
I'd say it's a left over for the other scripts. Here I've only kept the pattern of the other scripts. Do you see any reason not to use `-x` here, but use it for `install_*.sh`?

About `-u`, considering that one wouldn't use the script if `$@` is empty, it might be interesting to include `-u` indeed.

Reply:
> I'd say it's a left over for the other scripts. Here I've only kept the pattern of the other scripts. Do you see any reason not to use -x here, but use it for install_*.sh?

At most that it would pollute the build output for the actual IOC images. But I hadn't considered that the other scripts had it already.

Reply:
Yeah, mostly just going through "downloading x..."

I'd be in favor of collapsing `tar -x` output, yeah.

Reply:
I've redirected the output from `wget` as well. Let me know if there is anything we can improve about that (maybe keeping the entire URL?).